src/web/handlers/GestureHandler.ts

```diff
@@ -736,8 +738,22 @@ export default abstract class GestureHandler {
     return false;
   }
 
+  public isButtonInConfig(mouseButton: MouseButton | undefined) {
+    return (
+      !mouseButton ||
+      (!this.config.mouseButton && mouseButton === MouseButton.LEFT) ||
+      (this.config.mouseButton && mouseButton & this.config.mouseButton)
+    );
+  }
+
   protected resetConfig(): void {}
 
+  public finalize(): void {
```

Comment:
Yes, it can be pretty misleading. 

Changed in [1da7bfb](https://github.com/software-mansion/react-native-gesture-handler/pull/2676/commits/1da7bfbb71e81a90ab41800260a5ba91eb5b550e)